client/src/Book.tsx

```diff
@@ -38,16 +46,28 @@ export const Book = (): JSX.Element => {
       <ThemeProvider theme={config.theme} style={{ height: '100%' }}>
         <Stack styles={backgroundStyles(config.theme)}>
           <Header companyName={config.companyName} parentid={PARENT_ID} />
-          <LayerHost
-            id={PARENT_ID}
-            style={{
-              position: 'relative',
-              height: '100%'
-            }}
-          >
-            {config.microsoftBookingsUrl === BOOKINGS_SPECIMEN_URL ? <WarningBanner /> : <></>}
-            <iframe src={config.microsoftBookingsUrl} scrolling="yes" style={embededIframeStyles}></iframe>
-          </LayerHost>
+          {config.microsoftBookingsUrl ? (
+            <LayerHost
+              id={PARENT_ID}
+              style={{
+                position: 'relative',
+                height: '100%'
+              }}
+            >
+              {config.microsoftBookingsUrl === BOOKINGS_SPECIMEN_URL ? <WarningBanner /> : <></>}
+              <iframe src={config.microsoftBookingsUrl} scrolling="yes" style={embededIframeStyles}></iframe>
+            </LayerHost>
+          ) : (
```

Comment:
nit: For better readability let's extract these into two components, `BookingsPage` and `NoSchedulingPage`